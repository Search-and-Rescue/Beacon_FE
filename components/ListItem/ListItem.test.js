describe('ListItem', () => {
  it.skip('should', () => {

  })
})