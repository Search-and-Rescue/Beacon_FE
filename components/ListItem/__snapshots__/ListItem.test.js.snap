// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem should match the snapshot with all of the data passed through 1`] = `
<View
  style={Object {}}
>
  <View
    style={Object {}}
  >
    <Button
      title="Add"
    />
  </View>
</View>
`;
